### PR TITLE
Openldap One Click App

### PR DESCRIPTION
**Pull request by jacquant**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v2/apps/phpldapadmin.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                }
+            },
+            "php$$cap_appnameadmin": {
+                "image": "osixia/phpldapadmin:$$cap_phpldapadmin_version",
+                "containerHttpPort": "443",
+                "restart": "always",
+                "depends_on": ["$$cap_appname"],
+                "environment": {
+                    "PHPLDAPADMIN_TRUST_PROXY_SSL": true,
+                    "PHPLDAPADMIN_LDAP_HOSTS": "$$cap_appname.$$cap_root_domain",
```
Comment (githubsaturn) [inline public/v2/apps/phpldapadmin.json:26]: Wouldn't `srv-captain--$$cap_appname` work too? If it works it'll be a better alternative as it uses an internal connection.

File: public/v2/apps/phpldapadmin.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "documentation": "https://github.com/osixia/docker-openldap",
+    "description": "OpenLDAP with administration interface",
+    "displayName": "OpenLDAP + phpLDAPadmin",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+                "image": "osixia/openldap:$$cap_openldap_version",
+                "restart": "always",
+                "ports": ["389:389", "636:636"],
```
Comment (githubsaturn) [inline public/v2/apps/phpldapadmin.json:12]: Do ports really need to be exposed? If it's for internal use, you can access this by `srv-captain--$$cap_appname:389` without port mapping.

File: public/v2/apps/phpldapadmin.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        {
+            "id": "$$cap_openldap_ldap-organisation",
+            "label": "Ldap Organisation",
+            "description": "The Organization name",
+            "validRegex": "/.{1,}/"
+        },
+        {
+            "id": "$$cap_openldap_ldap-domain",
+            "label": "Ldap Host",
+            "description": "The host",
```
Comment (githubsaturn) [inline public/v2/apps/phpldapadmin.json:54]: Maybe provide an example in the description as the regex rule is quite complex to understand. Something like `myhost.domain.something` would work, right?

Comment (jacquant) [reply public/v2/apps/phpldapadmin.json:54]: Ok I'll do that! I take the advice into accounts for the next PRs :smile:

File: public/v2/apps/phpldapadmin.json (hunk 2 above)
Comment (jacquant) [reply public/v2/apps/phpldapadmin.json:12]: In my case, I needed it to be accessible from other machines. I can remove it and explain how to activate it in the end instructions.

Comment (githubsaturn) [reply public/v2/apps/phpldapadmin.json:12]: That'd be perfect! Thanks!

File: public/v2/apps/phpldapadmin.json (hunk 1 above)
Comment (jacquant) [reply public/v2/apps/phpldapadmin.json:26]: It works so I change :)

File: public/v2/apps/phpldapadmin.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            "$$cap_appname": {
+                "image": "osixia/openldap:$$cap_openldap_version",
+                "restart": "always",
+                "notExposeAsWebApp": "true",
+                "environment": {
+                    "LDAP_ORGANISATION": "$$cap_openldap_ldap-organisation",
+                    "LDAP_DOMAIN": "$$cap_root_domain",
+                    "LDAP_ADMIN_PASSWORD": "$$cap_openldap_ldap-admin-password"
+                }
+            },
```
Comment (githubsaturn) [inline public/v2/apps/phpldapadmin.json:18]: One last question - doesn't openldap require any persistent volume? With current implementation it'll lose all its data upon restart.

Comment (jacquant) [reply public/v2/apps/phpldapadmin.json:19]: Done :smile:

**Comment by githubsaturn** [comment]
Superb! Thanks!

**Pull request merged**